examples/events/EiffelArtifactDeployedEvent/simple.json

```diff
@@ -0,0 +1,22 @@
+{
+  "meta": {
+    "type": "EiffelArtifactDeployedEvent",
+    "version": "1.0.0",
+    "time": 1234567890,
+    "id": "aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeee4",
+    "source": {
+      "domainId": "example.domain"
+    }
+  },
```

Comment:
Travis is failing since we have no data section. which is mandatory according to the specification.

if you add 
data : {}, 
I think it should work.